staging/src/k8s.io/api/apps/v1beta2/types.go

```diff
@@ -157,6 +157,12 @@ type RollingUpdateStatefulSetStrategy struct {
 	// Default value is 0.
 	// +optional
 	Partition *int32 `json:"partition,omitempty" protobuf:"varint,1,opt,name=partition"`
+	// The maximum number of pods that can be unavailable during the update.
+	// Value can be an absolute number (ex: 5) or a percentage of desired pods (ex: 10%).
+	// Absolute number is calculated from percentage by rounding down.
+	// Defaults to 1.
```

Comment:
May it be zero?

Reply:
added a comment, it cannot be zero @lavalamp